RecoVertex/BeamSpotProducer/src/BeamFitter.cc

```diff
@@ -672,6 +676,20 @@ void BeamFitter::dumpTxtFile(std::string & fileName, bool append){
     }
   }//if bx results needed
   else {
+  
+    beamspot::BeamSpotContainer currentBS;
+    
+    currentBS.beamspot       = fbeamspot      ;
+    currentBS.run            = frun           ;
+    std::copy(fbeginTimeOfFit, fbeginTimeOfFit+32, currentBS.beginTimeOfFit);
+    std::copy(fendTimeOfFit  , fendTimeOfFit  +32, currentBS.endTimeOfFit  );
+    currentBS.beginLumiOfFit = fbeginLumiOfFit;
+    currentBS.endLumiOfFit   = fendLumiOfFit  ;
+    std::copy(freftime, freftime+2, currentBS.reftime);
+          
+    beamspot::dumpBeamSpotTxt(fileName, append, currentBS);
+        
+    /*
```

Comment:
is this commented out code needed?
Please remove or add comments inline in the code why the commented out block is relevant